libs/ui/src/lib/forms/Action/ArrowLeft.icon.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const ArrowLeft = () => (
+  <svg
+    width={24}
+    height={24}
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
```

Comment:
Não sei qual serviço você usa Thatá, mas esse aqui é bem legal:

https://www.smooth-code.com/open-source/svgr/playground/

Um ponto que eu acho interessante é que ele deixa props a mais como possibilidade de uso ou pra podermos subescrever alguma


```suggestion
const ArrowLeft = (props) => (
  <svg
    width={24}
    height={24}
    viewBox="0 0 24 24"
    fill="none"
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
```